Snakefile

```diff
@@ -289,7 +289,20 @@ rule build_bus_regions:
         "scripts/build_bus_regions.py"
 
 
+res_tech_names = list(config["renewable"].keys())
+config_cutouts = [
+    (lambda x: config["renewable"][x]["cutout"])(x) for x in res_tech_names
+] + list(config["atlite"]["cutouts"].keys())
```

Comment:
May you test if the following works?
(except pre-commit...)
```suggestion
config_cutouts = [d_value["cutout"] for tc, d_value in config["renewable"].items()] + list(config["atlite"]["cutouts"].keys())
```

Reply:
If yes, I think we could move this definition and the for loop in a function to be placed somewhere and then call the function here.
That may increase the readability of the script.
In the future, the auxiliary functions may be moved in a separate function as pypsa-eur did

Reply:
Wow, nice solution! Implemented :D